third_party/libvpx/source/libvpx/vp9/encoder/vp9_encoder.c

```diff
@@ -2086,12 +2086,12 @@ void vp9_change_config(struct VP9_COMP *cpi, const VP9EncoderConfig *oxcf) {
   if (last_w != cpi->oxcf.width || last_h != cpi->oxcf.height) {
     vpx_free(cpi->consec_zero_mv);
     CHECK_MEM_ERROR(
-        &cm->error, cpi->consec_zero_mv,
+        cm, cpi->consec_zero_mv,
```

Comment:
Why is this being changed?

Reply:
A bug fix a few weeks ago changed these lines https://github.com/youtube/cobalt/commit/7054654e43f3cd63c652be5d90d3a866a2c7bc4f#diff-3d2bf6451041190b5d73df60d7e17eb41d7cf8f41701a833eeee8dfcc2bebf37R2088-R2095. The call uses `CHECK_MEM_ERROR` from a newer version of the libvpx library than we have, it's what Chrome [currently uses](https://source.chromium.org/chromium/chromium/src/+/main:third_party/libvpx/source/libvpx/vp9/encoder/vp9_encoder.c;l=2164-2171;drc=fe5dc9f7fc2f98709184dace2308d51265f2800d). Making the change to have it build seemed better than reverting the PR or doing another library update. `CHECK_MEM_ERROR` will access `error` itself, rather than the caller passing `&cm->error`